shell/artifacts/Words/source/GamePane.js

```diff
@@ -221,11 +221,29 @@ defineParticle(({DomParticle, resolver}) => {
     _tilesToWord(tiles) {
       return tiles.map(t => t.letter).join('');
     }
+    willReceiveProps({renderParticle}, state) {
+      if (renderParticle && !state.renderParticleSpec) {
+        const renderParticleSpec = JSON.stringify(renderParticle.toLiteral());
+        const renderRecipe = DomParticle
```

Comment:
This seems pretty heavyweight.. Why does GamePane need to create on-the-fly inner recipes? Could ShowSingleStats be a suggestion (instead of embedded in GamePane) to leverage our standard composition?

Reply:
The theory is that Game and Post might set different recipes, though currently they're the same. Let's discuss the suggestion idea in person, I think we're wide open to alternate approaches in general, but in particular I think we need/want them to stick persistently on a per-item basis once that item is written.